mftf/2.3/reporting.md

```diff
@@ -0,0 +1,367 @@
+---
+---
+
+# Reporting
+
+The Magento Functional Testing Framework provides two types of reporting:
+
+- In CLI, when you run [`mftf`][mftf] or [`codecept`][codecept] commands.
+- In a web browser, using [Allure Framework][] after test run was complete.
+
+All the reporting artifacts are stored at the `dev/tests/acceptance/tests/_output` in the Magento root directory.
+The directory contains:
+
+- `allure-results/` that is a directory generated and served by the Allure Framework.
+- `failed` that is a text file containing relative paths to failed tests after the last test run.
+  The paths are relative to `dev/tests/acceptance/`.
+- `.html` and `.png` files that are screenshots of fails in HTML and PNG formats.
+  To cleanup the `_output/` directory, remove them manually.
+
+The `mftf` tool logs its output at `dev/tests/acceptance/mftf.log` continuously.
```

Comment:
The `mftf` tool logs output continuously to the `dev/tests/acceptance/mftf.log` file.

Reply:
:ok: